# Path to the folder that has the frozen_inference_graph.pb file in it
graph_path: "~/networks/ssd_mobilenet_v1.pb"

# File with all the label names in it
labels_path: "~/networks/tf_label_map.pbtxt"

# Topic to get the images from
camera_namespace: "/camera/rgb/image_rect_color"

# Threshold for what we'll decide is a real object
threshold: 0.5
